### PR TITLE
Bicaws 1642 hadolint

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1  # Use the ref you want to point at
```
Comment (jaskaransarkaria) [inline .pre-commit-config.yaml:5]: oh could this point to the newest version please `v4.1.0`

**Review by daviesjamie — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -23,9 +23,9 @@ RUN npm run build
 
 FROM ${BUILD_IMAGE} as runner
 
-RUN useradd nextjs
-RUN groupadd nodejs
-RUN usermod -a -G nodejs nextjs
+RUN useradd nextjs && \
```
Comment (jaskaransarkaria) [inline Dockerfile:26]: Not sure if it's useful but node images already ship with a non-root user https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#non-root-user

**Review by jaskaransarkaria — DISMISSED** [review]

File: Dockerfile (hunk 2 above)
Comment (brettminnie) [reply Dockerfile:26]: We use our own node image

File: .pre-commit-config.yaml (hunk 1 above)
Comment (brettminnie) [reply .pre-commit-config.yaml:5]: :+1:

**Review by jaskaransarkaria — APPROVED** [review]

**Review by daviesjamie — APPROVED** [review]

**Pull request merged**